src/App.tsx

```diff
@@ -16,35 +19,69 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.DEFAULT);
+  const [isReversed, setIsReversed] = useState(false);
+
+  const handleReset = () => {
+    setSortField(SortType.DEFAULT);
+    setIsReversed(false);
+  };
```

Comment:
The function's name 'handleReset' is good as it starts with a verb, but the 'handleClick' for sorting buttons (lines 43 and 53) could be improved. According to the checklist, function names should start with a verb to describe their purpose more clearly. Consider renaming the inline arrow functions for sorting buttons to something like 'handleSortAlphabetically' and 'handleSortByLength' respectively.